README.md

```diff
@@ -32,15 +32,25 @@ Depending on the device of you choice, install the GUI for controlling the softw
    - "muondetector-gui_1.1.2-ubuntu_bionic-x64.deb" on a Ubuntu 18.xx machine 
    - "muondetector-gui_1.1.2-windows-x64.zip" on a 64-bit Windows machine 
 
-### Installation from source (master branch)
-under development
+### Installation from source
+When compiling directly from the git repository, due to a change in build system with version [2.0.0](https://github.com/MuonPi/muondetector/tree/v2.0.0) the steps diverge before and after this update. Also note that you need to manually build and install the [paho-mqttpp](https://github.com/eclipse/paho.mqtt.cpp) library.
 
-When compiling the master branch, pre-installation of all dependencies is needed (see some dependencies in the section below). Clone the repository and follow the steps below:
+#### version < 2.0.0
+Versions before this commit use qmake as build system.
+Follow the steps below to build the daemon:
 1. In folder `./muondetector-shared` excecute `qmake` and `make`. Depending on error messages check for missing dependencies.
 2. Copy the compiled libraries to your library folder: `sudo cp ./muondetector-shared/bin/* /usr/lib/`
 3. In folder `./muondetector-daemon` excecute `qmake` and `make`. Depending on error messages check for missing dependencies.
 4. The daemon can be found in folder `./muondetector-daemon/bin/`
 
+#### version >= 2.0.0
+Versions after this commit use CMake as build system. The steps to building the daemon are as follows:
+1. enter the build directory
+2. run `cmake ../source -DMUONDETECTOR_BUID_GUI=OFF`
+3. run `make package`
+4. go to `../packages`

```

Comment:
Confusion because now you are in build directory and packages should also be in build directory so you don't need ../ !?

Reply:
fixed